packages/dev/core/src/Meshes/abstractMesh.hotSpot.ts

```diff
@@ -93,16 +93,28 @@ export function GetTransformedPosition(mesh: AbstractMesh, index: number, res: V
  * @param mesh mesh used to get hotspot from
  * @param hotSpotQuery point indices and barycentric
  * @param res output world position
+ * @param resNormal optional output world normal
  */
-export function GetHotSpotToRef(mesh: AbstractMesh, hotSpotQuery: HotSpotQuery, res: Vector3): void {
+export function GetHotSpotToRef(mesh: AbstractMesh, hotSpotQuery: HotSpotQuery, res: Vector3, resNormal?: Vector3): void {

```

Comment:
Maybe rename the `res` param?
```suggestion
export function GetHotSpotToRef(mesh: AbstractMesh, hotSpotQuery: HotSpotQuery, resPosition: Vector3, resNormal?: Vector3): void {

```